doc/_templates/customindex.html

```diff
@@ -40,7 +40,14 @@ <h2>Open Standard</h2>
   <div>
     <h2>Our Mission</h2>
     <p>
-      End-to-end encrypted e-mail has been around for decades, but has failed to see wide adoption outside of specialist communities.  Autocrypt aims to provide encryption convenient enough for much wider adoption. Through strengthening the e-mail ecosystem step-by-step, we want to increase user autonomy and privacy for everyone.
+      Autocrypt aims to provide end-to-end encrypted e-mail that is
+      convenient enough for wide adoption.
+    </p>
+    <p>
```

Comment:
I agree with the shuffling around but less with making it into three instead of two paras.  Would you mind removing the extra para and just have the second sentence next the first?